decodable/client/client.py

```diff
@@ -47,6 +49,51 @@ def from_dict(cls, response: Dict[str, Any]) -> PreviewResponse:
         )
 
 
+@dataclass
+class AccountInfoResponse:
+    subscription: SubscriptionResponse
+    deployment_type: str
+    data_plane_hostname: str
+    uses_new_secrets: bool
+
+    @classmethod
+    def from_dict(cls, response: Dict[str, Any]) -> AccountInfoResponse:
+        return cls(
+            subscription=SubscriptionResponse.from_dict(response["subscription"]),
+            deployment_type=response["deployment_type"],
+            data_plane_hostname=response["data_plane_hostname"],
+            uses_new_secrets=response["uses_new_secrets"],
```

Comment:
fwiw, I think some standard library should handle these standard conversions. The [jsons library](https://pypi.org/project/jsons/) can in fact do this, but I didn't want to introduce it and use it only for a few special cases. If there is consensus that this is a good idea, we can add it later and convert all such usages to `jsons`